rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_motor_model.h

```diff
@@ -46,13 +47,16 @@ const static int CCW = 1;
 const static int CW = -1;
 } // namespace turning_direction
 
-namespace gazebo {
+enum motor_type {
+  VELOCITY = 1,
+  POSITION = 2,
+  FORCE = 3
+};
 
-// Default values
-//static const std::string kDefaultCommandSubTopic = "gazebo/command/motor_speed";
-//static const std::string kDefaultWindSpeedSubTopic = "gazebo/wind_speed";
+namespace gazebo {
 
-typedef const boost::shared_ptr<const gz_mav_msgs::CommandMotorSpeed> GzCommandMotorSpeedMsgPtr;
+// Changed name from speed to input for more generality. TODO: integrate general actuator command.
```

Comment:
TODO(kajabo): Integrate ...
In case someone wonders what this todo meant, they know who to ask :)